charts/bc-wallet/templates/showcase-creator/service.yaml

```
@@ -0,0 +1,24 @@
+{{- if .Values.showcase_creator.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-{{ .Values.showcase_creator.name }}
+  namespace: {{ .Values.global.namespaceOverride | default .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ .Values.showcase_creator.name }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    {{- if .Values.showcase_creator.labelOverride }}
+    app.kubernetes.io/component: {{ .Values.showcase_creator.labelOverride }}
```

```suggestion
    app.kubernetes.io/component: {{ .Values.showcase_creator.labelOverride }}
```
kinda weird the combination of pascalCase and snake_case

---

yea, that's normal. showcase_creator is the app name (usually snake_case) and overides are the values (pascal)